CGMBLEKit Example/AppDelegate.swift

```diff
@@ -106,3 +140,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate, TransmitterDelegate {
         }
     }
 }
+
+struct CommandQueue {
```

Comment:
This doesn't provide anything on top of `Array<Command>`, and it isn't thread-safe to use. You might consider making this type a class, and internally thread-safe using `os_unfair_lock`